package.json

```diff
@@ -78,8 +79,7 @@
 		"mocha": "^9.1.1",
 		"nyc": "^15.1.0",
 		"prettier": "^2.4.1",
-		"ts-expose-internals": "=4.5.5",
-		"ttypescript": "^1.5.12",
+		"ttypescript": "^1.5.13",
```

Comment:
Didn't mean to do this, probably got done by npm somewhere in the conflict resolution with the beta versions. I guess it's fine to stay?